products/BellHybrid/alarms/src/actions/FrontlightAction.cpp

```diff
@@ -103,11 +106,25 @@ namespace alarms
         default:
             break;
         }
+
         return pimpl->turnOff();
     }
 
+    screen_light_control::ManualModeParameters ManualFrontlightAction::prepareParameters()
+    {
+        std::string brightnessString =
+            settings.getValue(bell::settings::Alarm::brightness, settings::SettingsScope::Global);
+        float brightness = 50.0f;
+        if (!brightnessString.empty()) {
+            brightness = frontlight_utils::fixedValToPercentage(std::stoi(brightnessString));
+        }
+        return screen_light_control::ManualModeParameters{.manualModeBrightness = brightness};
```

Comment:
Please avoid designated _initializers_: they're perfectly allowed only in C (since C99) or since C++20.

Reply:
I see that GCC supports `designated initializers` in C++17. However, it is not part of the standard in C++17 so I removed it from the code. :)